__tests__/app.test.js

```diff
@@ -356,4 +356,22 @@ describe("/api/comments/:comment_id", () => {
             expect(body.message).toBe('bad request')
         })
     })
+});
+
+
+describe("GET /api/users", () => {
+    test("GET 200: responds with an array of all users", () => {
+        return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({body}) => {
+            const {users} = body;
```

Comment:
Good assertions - not necessary to repeat "GET" in the test description when it's in the describe block, otherwise test descriptions are nice and clear